src/types/redis_stream.cc

```diff
@@ -232,6 +241,34 @@ std::string Stream::encodeStreamConsumerMetadataValue(const StreamConsumerMetada
   return dst;
 }
 
+StreamConsumerMetadata Stream::decodeStreamConsumerMetadataValue(const std::string &value) {
+  StreamConsumerMetadata consumer_metadata;
+  rocksdb::Slice input(value);
+  GetFixed64(&input, &consumer_metadata.pending_number);
+  GetFixed64(&input, &consumer_metadata.last_idle);
+  GetFixed64(&input, &consumer_metadata.last_active);
+  return consumer_metadata;
+}
+
+StreamSubkeyType Stream::identifySubkeyType(const rocksdb::Slice &key) {
+  InternalKey ikey(key, storage_->IsSlotIdEncoded());
+  Slice subkey = ikey.GetSubKey();
+  if (subkey.size() <= 16) {  // id.ms(uint64) + id.seq(uint64)
```

Comment:
If you can introduce constants with descriptive names for the values `8`, `16`, and another `8`, you will be able to get rid of explanation comments in the code.

Reply:
Hi @Yangsx-1, can you resolve this comment?

Reply:
Yes, i'll remove these comments in the code and add some const variables.

Reply:
That's cool.